contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import psutil
+
+from opencensus.metrics.export.gauge import DerivedLongGauge
+from opencensus.metrics.export.standard_metrics import BaseStandardMetric
+from opencensus.metrics.export.standard_metrics import StandardMetricsProducer
+
+
+class StandardMetricsType(object):
+    AVAILABLE_MEMORY = "\\Memory\\Available Bytes"
+
+
+# Definitions taken from psutil docs
+# https://psutil.readthedocs.io/en/latest/
+class AvailableMemoryStandardMetric(BaseStandardMetric):
+    """ Metric for Available Memory
+        Avaliable memory is defined as memory that can be given
+        instantly to processes without the system going into swap
+    """
+    def __init__(self):
+        super(AvailableMemoryStandardMetric, self).__init__()
+
+    def register(self, registry):
+        available_memory_gauge = DerivedLongGauge(
+            StandardMetricsType.AVAILABLE_MEMORY,
+            'Amount of available memory in bytes',
+            'byte',
+            [])
+        available_memory_gauge.create_default_time_series(
+            self.get_available_memory)
+        registry.add_gauge(available_memory_gauge)
+
+    def get_available_memory(self):
```

Comment:
You might want to lift this out of the class too (or make it a static/classmethod) since it doesn't touch class state.